qemu/tests/uefi_check_rng_device.py

```diff
@@ -0,0 +1,36 @@
+import re
+
+from virttest import utils_misc
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Check virtio-rng device info in bios log:
+    1) boot guest with virtio-rng device
+    2) check info in ovmf log
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    def info_check(info):
+        """
+        Check log info
+        """
+        logs = vm.logsessions['seabios'].get_output()
+        result = re.search(info, logs, re.S)
+        return result
```

Comment:
How about `return re.search(info, logs, re.S)`?

Reply:
Ok, it is more reasonable, thanks!